### ISSUE TITLE
ERROR: Assigning objects of class Bignum is not supported.

### ISSUE THREAD

**Issue by aigjermo** [scrapper/postrunner#16]
I installed this today and am getting the following output for every command I try:
```
$ postrunner
ERROR: Assigning objects of class Bignum is not supported. Only PEROBS objects or one of the following classes are supported: NilClass, Integer, Float, String, Time, TrueClass, FalseClass
/home/anders/.gem/ruby/2.3.0/gems/perobs-4.1.0/lib/perobs/Log.rb:107:in `fatal'
/home/anders/.gem/ruby/2.3.0/gems/perobs-4.1.0/lib/perobs/ObjectBase.rb:226:in `_check_assignment_value'
/home/anders/.gem/ruby/2.3.0/gems/perobs-4.1.0/lib/perobs/Hash.rb:127:in `[]='
/home/anders/.gem/ruby/2.3.0/gems/perobs-4.1.0/lib/perobs/Store.rb:289:in `[]='
/home/anders/.gem/ruby/2.3.0/gems/postrunner-0.11.0/lib/postrunner/Main.rb:65:in `main'
/home/anders/.gem/ruby/2.3.0/gems/postrunner-0.11.0/lib/postrunner.rb:24:in `<module:PostRunner>'
/home/anders/.gem/ruby/2.3.0/gems/postrunner-0.11.0/lib/postrunner.rb:22:in `<top (required)>'
/usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/home/anders/.gem/ruby/2.3.0/gems/postrunner-0.11.0/bin/postrunner:4:in `<top (required)>'
/home/anders/.gem/ruby/2.3.0/bin/postrunner:22:in `load'
/home/anders/.gem/ruby/2.3.0/bin/postrunner:22:in `<main>'

*******************************************************************************
You have triggered a bug in PostRunner 0.11.0!
```

The only command I found which does not give this error is `postrunner --help`.
Maybe a version mismatch?

Here are my installed gems:
```
bigdecimal (1.2.8)
bindata (2.3.0)
did_you_mean (1.0.0)
fit4ruby (3.2.0)
io-console (0.4.5)
json (1.8.3)
mini_portile2 (2.4.0)
minitest (5.9.0)
net-telnet (0.1.1)
nokogiri (1.10.3)
perobs (4.1.0)
postrunner (0.11.0)
power_assert (0.2.7)
psych (2.1.0)
rake (10.5.0)
rdoc (4.2.1)
```

Installed with `gem install --user postrunner`. I did not have any ruby gems (besides the defaults) installed before this. Running debian stretch if that matters.

**Comment by aigjermo** [comment]
Quick update on this. After upgrading from stretch to buster and installing on ruby 2.5, things appear to be working fine. So it seems to be limited to ruby 2.3 (and maybe earlier versions).

**Comment by scrapper** [comment]
Thanks for calling this out. I've tested it on Ruby 2.1.5, 2.1.9 and 2.4.1 without issues. But the handling of Bignum changed from 2.3 to 2.4. I've updated the Gemspec now to require at least Ruby 2.4. That was only released 2.5 years ago, but there is very little I can do about it without breaking compatibility with the PEROBS databases out there.

**Comment by aigjermo** [comment]
Great! And btw, thanks for creating this software in the first place :-)
